test/units/plugins/filter/test_uuid.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible.plugins.filter.core import to_uuid
+
+
+UUID_DEFAULT_NAMESPACE_TEST_CASES = (
+    ('example.com', 'ae780c3a-a3ab-53c2-bfb4-098da300b3fe'),
+    ('test.example', '8e437a35-c7c5-50ea-867c-5c254848dbc2'),
```

Comment:
Let's also test non-ASCII characters
```suggestion
    ('test.example', '8e437a35-c7c5-50ea-867c-5c254848dbc2'),
    ('café.example', '8a99d6b1-fb8f-5f78-af86-879768589f56',),
```